src/Compatibility/Microsoft.DotNet.PackageValidation/Validators/PackageValidatorOption.cs

```diff
@@ -5,16 +5,50 @@
 
 namespace Microsoft.DotNet.PackageValidation.Validators
 {
-    public struct PackageValidatorOption
+    /// <summary>
+    /// A package validator option bag that is passed into package validators for configuration.
+    /// </summary>
+    public readonly struct PackageValidatorOption
```

Comment:
NIT: when you make the struct readonly, then I believe you don't need to have members explicitly say they are readonly, as this is done by the compiler.